### ISSUE TITLE
Am I doing something stupid?

### ISSUE THREAD

**Issue by royjr** [Spomky-Labs/otphp#81]
| Q                    | A
| -------------------- | -----
| Bug report?          | maybe
| Feature request?     | no
| BC Break report?     | no
| RFC? / Specification | yes
| Library version      | 9.0

Code..
```
<?php
use OTPHP\TOTP;
$otp = TOTP::create();
```

Error..
`Fatal error: Uncaught Error: Call to undefined method OTPHP\TOTP::create() in /Applications/XAMPP/xamppfiles/htdocs/1log/test.php:7 Stack trace: #0 {main} thrown in /Applications/XAMPP/xamppfiles/htdocs/1log/test.php on line 7`

Am I doing something stupid? Thanks

PHP version.. 7.1.7

Composer Latest Version..
```
Using version ^9.0 for spomky-labs/otphp
  - Installing christian-riesen/base32 (1.3.1) Loading from cache
  - Installing beberlei/assert (v2.7.6) Loading from cache
  - Installing spomky-labs/otphp (v9.0.0) Downloading: 100%    
```

**Comment by Spomky** [comment]
It looks like you forgot to include the `autoload.php` file generated by composer (see in the `vendor` directory).

```php
<?php

include_once(__DIR__.'/vendor/autoload.php');

use OTPHP\TOTP;
$otp = TOTP::create();
```

**Comment by royjr** [comment]
Silly me.. I actually had the autoload, but forgot to include that here..
`require 'vendor/autoload.php';`
and
`include_once(__DIR__.'/vendor/autoload.php');`

still same error.. 
`Fatal error: Uncaught Error: Call to undefined method OTPHP\TOTP::create() ...`

The following works fine for me..
`$totp = new TOTP('test',$mySecret,30,'sha1',6);`

So it seems as if your changes to 9.0 are not being recognized. To help with some background info.. I used composer to install before realizing I only had php7.0.. so I reinstalled XAMPP completely, verified its using the right php directory and that it was the correct version.. composer update.. and this is where im stuck at. oh wait!

found this in composer's installed.json..
`
"name": "spomky-labs/otphp",
"version": "v8.3.0",
"version_normalized": "8.3.0.0",
`

Yes.. I was doing something stupid. So.. I used a symbolic link before upgrading XAMPP.. and I imported a new htdocs directory.. so I upgraded composer in my backup directory =/

Well, all seems to be working now. Great Work!
